jobs/build.Jenkinsfile

```
@@ -230,9 +230,38 @@ lock(resource: "build-${params.STREAM}") {
 
         // fetch from repos for the current build
         stage('Fetch') {
+            def overrides_fetch_param = ""
+            # Dont run this for production builds
+            recent_commits_to_lockfiles() {
+                // Checking if last commits meet the criteria
+                shwrap("""
```

I think we need to use `shwrapRc` here and then check for `0` or `1`. See other examples from this repo of using `shwrapRc`. 
```suggestion
                shwrapRc("""
```

---

Thanks for the above - can't find any documentation on the shwrap | shwrapRc, got any links?